NuGetPackageVerifier.json

```diff
@@ -19,49 +19,235 @@
     ],
     "packages": {
       "Microsoft.NET.Sdk.Razor": {
-        "exclusions": {
+        "Exclusions": {
           "ASSEMBLY_DESCRIPTION": {
             "tools/netcoreapp2.0/Microsoft.CodeAnalysis.CSharp.dll": "This assembly is not owned by us and does not follow our conventions.",
-            "tools/netcoreapp2.0/Microsoft.CodeAnalysis.dll": "This assembly is not owned by us and does not follow our conventions."
+            "tools/netcoreapp2.0/Microsoft.CodeAnalysis.dll": "This assembly is not owned by us and does not follow our conventions.",
+            "tools/netcoreapp2.0/cs/Microsoft.CodeAnalysis.CSharp.resources.dll": "This assembly is not owned by us and does not follow our conventions.",
```

Comment:
So this is pretty unfortunate. Currently the `Razor.SDK` is **fat** package in that it contains publish output. We have a work item tracking improving this.

But yeah, Roslyn added their resources dlls, and so they started showing up here. This is the only thing I could think of to do.